modules/platforms/dotnet/Apache.Ignite.Internal.ComputeExecutor/SystemInfoJob.cs

```diff
@@ -0,0 +1,42 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Apache.Ignite.Internal.ComputeExecutor;
+
+using System.Diagnostics;
+using Ignite.Compute;
+
+/// <summary>
+/// A built-in compute job that provides system information about the executor.
+/// Useful for .NET executor troubleshooting.
+/// </summary>
+public sealed class SystemInfoJob : IComputeJob<object?, string>
```

Comment:
Would this job pose a vulnerability in production? Maybe we should disable it by default?

Reply:
We do not do anything sensitive here - do you see any potential problem?

Reply:
Removed this job to be on the safe side.